src/EnergyPlus/EvaporativeCoolers.cc

```diff
@@ -4660,104 +4526,210 @@ namespace EvaporativeCoolers {
-                        SimEvapCooler(ZoneEvapUnit(UnitNum).EvapCooler_1_Name, ZoneEvapUnit(UnitNum).EvapCooler_1_Index);
-                    }
+        // PURPOSE OF THIS SUBROUTINE:
+        // caculates zone evaporative cooler sensible and latent outputs
 
-                    if ((ZoneEvapUnit(UnitNum).EvapCooler_2_Index > 0) && ZoneEvapUnit(UnitNum).EvapCooler_2_AvailStatus) {
-                        SimEvapCooler(ZoneEvapUnit(UnitNum).EvapCooler_2_Name, ZoneEvapUnit(UnitNum).EvapCooler_2_Index);
-                    }
-                    if (ZoneEvapUnit(UnitNum).FanLocation == DrawThruFan) {
-                        if (ZoneEvapUnit(UnitNum).FanType_Num != DataHVACGlobals::FanType_SystemModelObject) {
-                            Fans::SimulateFanComponents(
-                                ZoneEvapUnit(UnitNum).FanName, false, ZoneEvapUnit(UnitNum).FanIndex, _, ZoneCompTurnFansOn, ZoneCompTurnFansOff);
-                        } else {
-                            HVACFan::fanObjs[ZoneEvapUnit(UnitNum).FanIndex]->simulate(_, ZoneCompTurnFansOn, ZoneCompTurnFansOff, _);
-                        }
-                    }
-                } else {
-                    Node(ZoneEvapUnit(UnitNum).OAInletNodeNum).MassFlowRate = 0.0;
-                    Node(ZoneEvapUnit(UnitNum).OAInletNodeNum).MassFlowRateMaxAvail = 0.0;
-                    Node(ZoneEvapUnit(UnitNum).FanInletNodeNum).MassFlowRate = 0.0;
-                    Node(ZoneEvapUnit(UnitNum).FanInletNodeNum).MassFlowRateMaxAvail = 0.0;
-                    Node(ZoneEvapUnit(UnitNum).FanOutletNodeNum).MassFlowRate = 0.0;
-                    Node(ZoneEvapUnit(UnitNum).FanOutletNodeNum).MassFlowRateMaxAvail = 0.0;
-                    Node(ZoneEvapUnit(UnitNum).UnitOutletNodeNum).MassFlowRate = 0.0;
-                    Node(ZoneEvapUnit(UnitNum).UnitOutletNodeNum).MassFlowRateMaxAvail = 0.0;
-
-                    Node(EvapCond(ZoneEvapUnit(UnitNum).EvapCooler_1_Index).InletNode).MassFlowRate = 0.0;
-                    Node(EvapCond(ZoneEvapUnit(UnitNum).EvapCooler_1_Index).InletNode).MassFlowRateMaxAvail = 0.0;
-                    Node(EvapCond(ZoneEvapUnit(UnitNum).EvapCooler_1_Index).OutletNode).MassFlowRate = 0.0;
-                    Node(EvapCond(ZoneEvapUnit(UnitNum).EvapCooler_1_Index).OutletNode).MassFlowRateMaxAvail = 0.0;
-
-                    if (ZoneEvapUnit(UnitNum).EvapCooler_2_Index > 0) {
-                        Node(EvapCond(ZoneEvapUnit(UnitNum).EvapCooler_2_Index).InletNode).MassFlowRate = 0.0;
-                        Node(EvapCond(ZoneEvapUnit(UnitNum).EvapCooler_2_Index).InletNode).MassFlowRateMaxAvail = 0.0;
-                        Node(EvapCond(ZoneEvapUnit(UnitNum).EvapCooler_2_Index).OutletNode).MassFlowRate = 0.0;
-                        Node(EvapCond(ZoneEvapUnit(UnitNum).EvapCooler_2_Index).OutletNode).MassFlowRateMaxAvail = 0.0;
-                    }
+        // METHODOLOGY EMPLOYED:
+        // na
 
```

Comment:
We don't add unused boiler plate comments anymore to new functions.

Reply:
Done